bogota-abril-2019/css-magazine-cover/marvin-cely/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>My Magazine</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="reset.css">
+    <link rel="stylesheet" type="text/css" href="main.css">
+</head>
+
+<body>
+    <main>
+        <header>
+            <h1>
+                VOGUE
+            </h1>
+            <p>©</p>
+            <p>FOTOJET</p>

```

Comment:
Unnecessary used of Capital letter. Please use Text-transform CSS property.

https://css-tricks.com/almanac/properties/t/text-transform/

Reply:
Capital letter with Text-transform CSS property reviewed